src/action_runner/resource.h

```diff
@@ -1,3 +1,10 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by action_runner.rc
+
+#define FILE_DESCRIPTION "PowerToys ActionRunner"
+#define INTERNAL_NAME "action_runner"
+#define ORIGINAL_FILENAME "action_runner.exe"
```

Comment:
We area going to localize these strings, it's better to add a comment to indicate it explicitly.